### ISSUE TITLE
Draw behind navigation bar on Android 10

### ISSUE THREAD

**Issue by semanticer** [toggl/mobileapp#6113]
## Design Improvement Proposal
 it is now strongly recommended for apps to draw behind the navigation bar when running on Android Q+ for a more compelling and modern UX. https://medium.com/androiddevelopers/gesture-navigation-going-edge-to-edge-812f62e4e83e

Basically we want to achieve this: https://miro.medium.com/max/1200/0*5Eqz13ClzwZlKLMV 

## Dependencies
Hopefully - we don't need AndroidX migration for this one

**Comment by amulware** [comment]
How is this gonna work with our bottom navigation and timer bars? Should a designer get involved here?

**Comment by semanticer** [comment]
Our navigation bar under the tab bar actually already looks as it should by accident but we have small issues in other views where we create an unnecessary white bar in the navigation area. I don't think we need a designer for this, it should be pretty straightforward if there is no secret Xamarin problem as usual. I'm for moving this to 2.6 if @kornelijepetak is ok with it.

**Comment by semanticer** [comment]
Ok guys, it looks like we can add this one to our long list of things we are not able to do in Xamarin. It took me around 20 min implementing this in pure-android side-project and I don't even want to disclose how much time I wasted doing this in our project. In the end, I have no idea why but wasn't able to get the correct behavior around the navbar. [Here is my WIP](https://github.com/toggl/mobileapp/tree/wip-draw-behind-nav-bar) branch where I isolated the problem into new TestActivity. 
Btw, I was able to simplify our day/night theme structure as a side effect so I'm gonna submit this as separate PR. We should move this somewhere or close completely @daividssilverio :-(
